garde/tests/ui/compile-fail/length_bad_equal_combined.rs

```diff
@@ -0,0 +1,9 @@
+#![allow(dead_code)]
+
+#[derive(garde::Validate)]
+struct Test<'a> {
+    #[garde(length(min = 1, equal = 10))]
+    field: &'a str,
+}
```

Comment:
This could use a few additional cases:

```suggestion
struct Test<'a> {
    #[garde(length(min = 1, equal = 10))]
    min_equal: &'a str,
    #[garde(length(max = 1, equal = 10))]
    max_equal: &'a str,
    #[garde(length(min = 1, max = 1, equal = 10))]
    min_max_equal: &'a str,
}
```

Reply:
Sure thing